spec/event_context.rb

```diff
@@ -2,7 +2,7 @@
 
 shared_context "event" do
   let(:table_id) { 1 }
-  let(:rows) { double("List") }
+  let(:rows) { Java::JavaUtil::LinkedList.new(Array.new(1) { Array.new(2) })}
```

Comment:
same comment here as in https://github.com/twingly/ecco/pull/57/files#r515003860, looks like a mixture between Java and Ruby objects which probably do not reflect real-life

Reply:
Fixed here: 186eecbac3b2e4631996eb515e627839a6c0e8b8